test/claim-sol.ts

```diff
@@ -0,0 +1,211 @@
+const { expect } = require("chai");
+import { BigNumber } from "@ethersproject/bignumber";
+import { Result, RLP } from "ethers/lib/utils";
+import {
+  encodeGuaranteeData,
+  MAGIC_VALUE_DENOTING_A_GUARANTEE,
+} from "../ts/nitro-types";
+import { Exit } from "../ts/types";
+const { ethers } = require("hardhat");
+import { Nitro } from "../typechain/Nitro";
+
+describe("claim (solidity)", function () {
+  let nitro: Nitro;
+
+  before(async () => {
+    nitro = await (await ethers.getContractFactory("Nitro")).deploy();
+
+    await nitro.deployed();
+  });
+
+  const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+  const A_ADDRESS = "0x96f7123E3A80C9813eF50213ADEd0e4511CB820f";
+  const B_ADDRESS = "0x53484E75151D07FfD885159d4CF014B874cd2810";
+
+  const initialOutcome: Exit = [
+    {
+      asset: ZERO_ADDRESS,
+      data: "0x",
+      allocations: [
+        {
+          destination: A_ADDRESS,
+          amount: "0x05",
+          callTo: ZERO_ADDRESS,
+          data: "0x",
+        },
+        {
+          destination: B_ADDRESS,
+          amount: "0x05",
+          callTo: ZERO_ADDRESS,
+          data: "0x",
+        },
+      ],
+    },
+  ];
+
+  const guarantee: Exit = [
+    {
+      asset: ZERO_ADDRESS,
+      data: "0x",
+      allocations: [
+        {
+          destination: ZERO_ADDRESS, // TODO: What should this be?
```

Comment:
Should the destination be set to the target channel Id?

Reply:
87412d7